### PR TITLE
Fix tailer path in README

### PR DESCRIPTION
**Pull request by jcrosby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ shipped with deployinator
 - Run the tailer as a background service (using whatever init flavor you like)
 
 ```sh
-    ./bin/deployinator-tailer.rb &
+    ./deployinator/deployinator-tailer.rb &
```
Comment (jayson) [inline README.md]: Did you run bundle install --binstubs deployinator? That should put the file in bin. Are you perhaps not using bundler or have a different config for it?

Comment (dwradcliffe) [inline README.md]: @jayson the binstub command is wrong, see #48

Comment (jayson) [inline README.md]: Merged. After running the binstub command, did you still need to use this pull request?

**Comment by jayson** [comment]
This is working for me with the new binstubs command, closing. Please reopen with more details if this doesn't resolve it for you

**Pull request closed without merging**